Tag create session metrics with response cause

ebengt opened the pull request:
Make it possible to see accepted/rejected session create request
by adding response cause in metrics.

RoadRunnr commented on test/ggsn_SUITE.erl:
```diff
@@ -1310,3 +1344,24 @@ load_ocs_config(Initial, Update) ->
+	%% Remove the ones that have not changed.
+	Value_before = socket_counter_metrics_ok_value( Metrics_before -- Metrics_after, Path_part ),
+	Value_after = socket_counter_metrics_ok_value( Metrics_after -- Metrics_before, Path_part ),
+	1 = Value_after - Value_before.
```
please use ?equal(...) for this.

RoadRunnr commented:
LGTM

ebengt commented:
Example of Prometheus output. The new entry is in the middle. Due to problems with config I can only get response with this one kind of error.
...deleted a lot
# TYPE socket_gtp_c_epc_tx_v1_create_pdp_context_response_retransmit counter
socket_gtp_c_epc_tx_v1_create_pdp_context_response_retransmit 0

# TYPE socket_gtp_c_epc_tx_v1_create_pdp_context_response_user_authentication_failed_count counter
socket_gtp_c_epc_tx_v1_create_pdp_context_response_user_authentication_failed_count 2

# TYPE socket_gtp_c_epc_tx_v1_data_record_transfer_request_count counter
socket_gtp_c_epc_tx_v1_data_record_transfer_request_count 0
...deleted a lot
